Workbooks/WebApplicationFirewallFirewallEvents.json

```diff
@@ -280,602 +407,236 @@
         "size": 0,
-        "exportParameterName": "Message",
-        "exportDefaultValue": "{ \"Name\":\"\", \"Type\":\"*\"}",
-        "exportToExcelOptions": "visible",
-        "title": "Attack events, by time",
+        "title": "Attack messages of IP address",
+        "noDataMessage": "Filtered messages are not attack events",
         "timeContext": {
           "durationMs": 0
         },
-        "timeContextFromParameter": "TimeRange",
+        "timeContextFromParameter": "timeBrushLowerSection",
+        "showExportToExcel": true,
         "queryType": 0,
         "resourceType": "microsoft.operationalinsights/workspaces",
-        "visualization": "barchart"
-      },
-      "customWidth": "70",
-      "name": "query - 14"
-    },
-    {
-      "type": 3,
-      "content": {
-        "version": "KqlItem/1.0",
-        "query": "AzureDiagnostics\r\n| where ResourceType == \"APPLICATIONGATEWAYS\" and (\"{WAF:lable}\" == \"All\" or Resource in ({WAF}))\r\n| where Message contains \"SQL Injection\" \r\n| summarize count() by hostname_s, Message\r\n| order by count_ desc ",
-        "size": 0,
-        "exportToExcelOptions": "visible",
-        "title": "SQL injection, by host name",
-        "timeContext": {
-          "durationMs": 0
-        },
-        "timeContextFromParameter": "TimeRange",
-        "queryType": 0,
-        "resourceType": "microsoft.operationalinsights/workspaces"
+        "visualization": "table",
+        "gridSettings": {
+          "filter": true
+        }
       },
-      "customWidth": "30",
-      "name": "query - 15"
+      "customWidth": "75",
+      "showPin": true,
+      "name": "query - 13"
     }
   ],
-  "styleSettings": {},
+  "fallbackResourceIds": [
+    "/subscriptions/6b1ceacd-5731-4780-8f96-2078dd96fd96/resourcegroups/soc-ns/providers/microsoft.operationalinsights/workspaces/soc-ns-logs"
+  ],

```

Comment:
This should not be here, as it will be a template for the community, this is a specific resource.